lib/index.lambda.js

```diff
@@ -228,21 +229,10 @@ const sendToSlack = async (message, channel, token) => {
   }
 };
 
-const sendToDiscord = async (message, channel, token) => {
+const sendToDiscordWebhook = async (message, webhook) => {
   try {
-    const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
-
-    client.once("ready", () => {
-      client.channels.fetch(channel).then(channel => {
-        channel.send(message).then(res => {
-          console.log("message sent.");
-          client.destroy();
-        });
-      });
-    });
-
-    // Log our bot in
-    client.login(process.env.DISCORD_BACKUP_SECRET);
```

Comment:
Note that the previous code would sometimes fail to send the message, because it didn't wait for the message to be sent before allowing the lambda to finish.